layouts/partials/footer.html

```diff
@@ -42,19 +42,21 @@ <h4>Sign Up for E-Newsletter</h4>
 					<li class="traveler"><a href="/travelers-info">Traveler's Info</a></li>
 					<li class="meetings"><a href="/meeting-planners">Meetings, Groups &amp; Conventions</a></li>
 					<li class="press"><a href="/press-media-industry">Press, Media &amp; Industry</a></li>
-					<li class="brochure"><a href="/contact/#contactform">Order Our<br>Visitor Guide<br>Today!</a></li>
+					<li class="guide"><a href="/contact/#contactform">Get Our Free Visitor Guide</a></li>
+					<li class="newsletter"><a href="/newsletter-archive">Newsletter Archive</a></li>
```

Comment:
I forgot to include this in the issue spec 👉  The client is not ready for the newsletter link yet. (Page in Forestry is blank and drafted). Could we wrap this in a `.GetPage`? The file is at `content/sitepages/newsletter-archive.md`